Formula/arangodb.rb

```diff
@@ -71,11 +74,28 @@ def install
       end
 
       system "cmake", "..", *args
-      system "make", "V=1", "Verbose=1", "VERBOSE=1", "install"
+      system "make", "install"
+
+      %w[arangod arango-dfdb arangosh foxx-manager].each do |f|
+        inreplace etc/"arangodb3/#{f}.conf", pkgshare, opt_pkgshare
+      end
     end
   end
 
   def post_install
+    oldpath = Regexp.escape("#{HOMEBREW_PREFIX}/Cellar/arangodb/3.0.") + "[12]"
```

Comment:
Mike's suggestion unfortunately won't quite work, as it's escaping too much. My suggestion would be to split this into two steps and then use `oldpath_regexp` further below (instead of repeatedly re-creating the regular expression on use):

```ruby
oldpath_prefix = "#{HOMEBREW_PREFIX}/Cellar/arangodb/3.0."
oldpath_regexp = /#{Regexp.escape(oldpath_prefix)}[12]/
```